src/components/SchedulePage/Timeslot.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import Event from '../SchedulePage/Event';
+import ScheduleRoundedIcon from '@material-ui/icons/ScheduleRounded';
+
+const useStyles = makeStyles(theme => ({
+	time: {
+		fontFamily: theme.typography.fontFamily
+	},
+	clockImage: {
+		marginTop: '1px',
+		marginBottom: '1px',
+		marginRight: '4px'
+	},
+	timeContainer: {
+		display: 'flex'
+	},
+	timeSlot: {
+		paddingLeft: '16px',
+		paddingRight: '16px',
+		[theme.breakpoints.up('md')]: {
+			width: theme.breakpoints.values.md,
+			marginLeft: 'auto',
+			marginRight: 'auto'
+		}
```

Comment:
There should be no need for this class if you had added a `<Container>` to `Schedule.js`, as I previously suggested.